bin/create

```diff
@@ -4,7 +4,8 @@ set -eu
 
 ROOT=$SNAP_COMMON/classic
 
-if [ -d $ROOT ]; then
+# check for the last bit that got created
+if [ -x $ROOT/usr/sbin/policy-rc.d ]; then
```

Comment:
Curious.. where does it come from?

Reply:
It is written at the end of the "create" step but zyga also flagged it so I simplify it again and will think about something smarter later.